src/main/scala/beam/router/skim/CountSkimmer.scala

```diff
@@ -0,0 +1,65 @@
+package beam.router.skim
+import beam.agentsim.infrastructure.taz.TAZ
+import beam.sim.BeamServices
+import beam.sim.config.BeamConfig
+import beam.sim.vehiclesharing.VehicleManager
+import com.typesafe.scalalogging.LazyLogging
+import org.matsim.api.core.v01.Id
+
+import scala.collection.immutable
+
+class CountSkimmer(beamServices: BeamServices, config: BeamConfig.Beam.Router.Skim.Skimmers$Elm)
+    extends AbstractSkimmer(beamServices, config) {
+  import CountSkimmer._
+
+  override val readOnlySkim: AbstractSkimmerReadOnly = CountSkims(beamServices)
```

Comment:
This and one other spot are still at risk of an initialization problem since not lazy. Was this run against the compiler flag `-Xcheckinit` and is ok?

Reply:
thanks, fixed!